all_documents.md

```diff
@@ -0,0 +1,20 @@
+# All Documents
+
+This page tracks the public documents that have been used for community discussions and development.
+
+## Current working documents
+
+* [Slide deck](https://docs.google.com/presentation/d/10jG8FYFAyKD8FI_MNo9RTqFUHg4A9e5jB_iHpA_Ut3A/edit?usp=sharing)
+* [Skills framework](./_data/skills.json)
+* [Example skill profiles](https://docs.google.com/spreadsheets/d/1M8f9luJrc3w4rz4lD-cEpJUfi6qigh9PBk7Fm5sq_jQ/edit?usp=sharing) (see tabs at the bottom)
+* [Resources for professional development](./_data/resources_list.csv)
+and the [original resource spreadsheet](https://docs.google.com/spreadsheets/d/1bHagH4hibAlPJ1inuaZMu-0tmNtjXoJeVPzGFwwWh0U/edit?usp=sharing) (not currently matching)
+* [Skill levels](https://docs.google.com/document/d/1syrY4gyaAcNuONK0ANUiHgHcbJ3P9RoqSjy57AjUsuM/edit#heading=h.nlkwx70wrmf)
+* [Issues on the rse-competencies-toolkit repository](https://github.com/RSEToolkit/rse-competencies-toolkit/issues)
+
+## No longer updated
```

Comment:
I didn't include the collaborative doc from RSECon23, as I'm not sure whether we can share this publicly.